peerdid/peer_did.py

```diff
@@ -26,21 +32,50 @@ def create_peer_did_numalgo_0(inception_key: PublicKeyAuthentication) -> PEER_DI
     :param inception_key: the key that creates the DID and authenticates when exchanging it with the first peer
     :return: generated peer_did
     """
-    pass
+    peer_did = 'did:peer:0z' + inception_key.encoded_value
+    if is_peer_did(peer_did):
+        return peer_did
+    else:
+        raise ValueError("Wrong Peer DID")
```

Comment:
If there any exceptions expected to be raised, it should be defined in the docs (`:raises`).